crates/turborepo-ui/src/tui/preferences.rs

```diff
@@ -1,132 +1,73 @@
-use std::{
-    fs::{self, File},
-    io::{BufReader, Write},
-};
-
 use serde::{Deserialize, Serialize};
-use serde_json::{from_reader, json, Value};
-use turbopath::AbsoluteSystemPathBuf;
-
-use super::task::TasksByStatus;
+use turbopath::{AbsoluteSystemPath, AbsoluteSystemPathBuf};
 
-#[derive(Serialize, Deserialize, Debug)]
-pub struct Preferences {
-    pub is_task_list_visible: Option<bool>,
-    pub active_task: Option<String>,
-    pub is_pinned_task_selection: Option<bool>,
+/// Structure for reading and writing preferences to disk
+pub struct PreferenceLoader {
+    file_path: AbsoluteSystemPathBuf,
```

Comment:
I think we might want to offer a non-persistent experience where we don't have an underlying file we read/write to. What do you think?

Reply:
I thought about this but it seemed like an unneeded knob if you really had all the controls that you need in hand. If you wanted to get back to the "without persistence" state, you could scroll to the top and tap "p". Maybe there's a missing bind for "Jump to top of tasks"?